src/sage/algebras/lie_algebras/verma_module.py

```diff
@@ -35,7 +35,80 @@
 from sage.rings.rational_field import QQ
 
 
-class VermaModule(CombinatorialFreeModule):
+class ModulePrinting(object):
```

Comment:
```suggestion
class ModulePrinting:
```
`object` not needed in Python 3